File: 240503/변수 값 교체하기 3/replacing-variable-values-3.py
# # <두 변수 값을 교환>

# # 1. temp 이용
# a, b = 5, 3

# temp = a
# a = b
# b = temp

# print(f"A is {a} B is {b}")

# # 2. , 를 이용하여 바로 교환하기
# a, b = 5, 3

# a, b = b, a

# print(f"A is {a} B is {b}")


# 문제
a, b = 3, 5
a, b = b, a

print(f"{a}\n{b}")

File: 250113/변수 값 교체하기 3/replacing-variable-values-3.cpp
// // <두 변수 값을 교환>
// temp = a;
// a = b;
// b = temp;

// #include <iostream>
// using namespace std;

// int main() {
//     int a = 5, b = 3;
//     int temp;

//     temp = a;
//     a = b;
//     b = temp;

//     cout << "A is " << a << " B is " << b;

//     return 0;
// }

// <문제: 변수 값 교체하기 3>
#include <iostream>
using namespace std;

int main() {
    int a = 3, b = 5;
    int temp = 0;

    temp = a;
    a = b;
    b = temp;

    cout << a << endl << b;
    return 0;
}